### PR TITLE
Update leon.json

### PR DESCRIPTION
**Pull request by leonskij**
Points `leon.json` to `leonskidev` instead of `leonskidev.github.io`.

### CODE REVIEW

File: domains/leon.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-  "repo": "https://github.com/LeonskiDev/leonskidev.github.io",
-  "description": "This is my portfolio website, if possible I'll also have bodged.is-a.dev as my blog.",
+  "repo": "https://github.com/LeonskiDev/leonskidev",
   "owner": {
     "username": "LeonskiDev",
     "email": "bodged@pm.me"
   },
   "record": {
-    "CNAME": "leonskidev.github.io"
+    "CNAME": "leonskidev.github.io/leonskidev/"
```
Comment (phenax) [inline domains/leon.json:8]: CNAME has to be a host name. If you're having trouble setting up your domain, [open up a discussion](https://github.com/is-a-dev/register/discussions) or create a support issue

**Review by phenax — CHANGES_REQUESTED** [review]

File: domains/leon.json (hunk 1 above)
Comment (LeonskiDev) [reply domains/leon.json:8]: Oh damn forgot about that, whoops. Thanks for reminding me.

**Comment by LeonskiDev** [comment]
Is there a way to make it point to https://leonskidev.github.io/leonskidev by any chance? I'd rather move it there if possible. If not then it's fine.

**Review by phenax — APPROVED** [review]

**Comment by phenax** [comment]
The changes have been published!! It should reflect in less than 24 hours.

## Here's what you need to do next
If your domain points to a server you own, add \`domain-name.is-a.dev\` to your server config. For https, you will have to configure ssl certificate to allow the new subdomain.

### For github pages users,
* Go to your github page repo (\`user/user.github.io\`)
* Open up the **settings** tab
* Scroll down to the **Github pages** section
* In the **Custom domain** text input, enter the domain you registered (\`domain-name.is-a.dev\`)
* Check the **Enforce HTTPS** checkbox below the input
* Give it some time to reflect and you should be good to go


## Need help with your domain?
If you are having trouble setting up your domain, [create an issue](https://github.com/is-a-dev/register/issues/new/choose). I will try my best to get back to you asap!


## Made a mistake in the record?
Don't worry, you can create a new pull request with the corrections


## Love/Hate the service?
**Love it?** Leave it a **star**! Also consider donating so that I can keep this service running forever!
**Hate it?** Please leave your feedback by [creating an issue](https://github.com/is-a-dev/register/issues/new/choose). I would really like to keep improving this service for other users.


## Wanna support this project?
Help me in my mission to keep this service alive forever by donating!

<a href="https://www.buymeacoffee.com/phenax" target="_blank">Buy Me A Coffee</a> <a href="https://liberapay.com/phenax" target="_blank">Liberapay recurring donation button</a>

**Comment by phenax** [comment]
> Is there a way to make it point to https://leonskidev.github.io/leonskidev by any chance? I'd rather move it there if possible. If not then it's fine.

Follow the instructions for github pages. The repo name doesn't really matter.

**Pull request merged**